src/TimeSteppers/quasi_adams_bashforth_2.jl

```diff
@@ -50,7 +50,7 @@ function QuasiAdamsBashforth2TimeStepper(grid, tracers,
     GT = typeof(Gⁿ)
     χ  = convert(FT, χ)
 
-    return QuasiAdamsBashforth2TimeStepper{FT, GT, IT}(χ, Inf, Gⁿ, G⁻, implicit_solver)
+    return QuasiAdamsBashforth2TimeStepper{FT, GT, IT}(χ, zero(FT), Gⁿ, G⁻, implicit_solver)
```

Comment:
I think we want to keep `Inf` because a `0` time-step is valid for some situations.